Capabilitystatements/capabilitystatement-client-ggz.xml

```diff
@@ -192,6 +192,20 @@
                 <name value="category"/>
                 <type value="token"/>
             </searchParam>
+        </resource>
+		<resource>
+            <type value="Condition" />
+            <interaction>
+                <code value="search-type" />
+            </interaction>
+            <searchParam>
+                <name value="patient" />
+                <type value="reference" />
+            </searchParam>
+            <searchParam>
+                <name value="category" />
+                <type value="token" />
+            </searchParam>
```

Comment:
Waarom staan deze searchParams erin? Die gebruiken we toch helemaal niet in het TO of zie ik iets over het hoofd?

Plus ik zie dat dit hele CapabilityStatement vol staat met searchParams die we niet gebruiken. Zie ik iets over het hoofd of is dit een nieuw ticket waarin we dat gaan corrigeren?

Reply:
Aangepast in overleg met Pieter ook. De searchparameters zijn eruit gehaald.